Tests/Models/BrickDimensionTests.swift

```diff
@@ -144,3 +155,63 @@ class BrickDimensionTests: XCTestCase {
     }
 
 }
+
+// MARK: - Utility methods to `fake` UIScreen.mainScreen
+extension BrickDimensionTests {
+
+    /// Setup UIScreen.mainScreen to call a different method
+    func swizzleScreen() {
+        NSObject.swizzleStaticMethodSelector(#selector(UIScreen.mainScreen), withSelector: #selector(UIScreen.stubMainScreen), forClass: UIScreen.self)
+    }
+
+    /// Reset UIScreen.mainScreen to call a different method
+    func resetScreen() {
+        NSObject.swizzleStaticMethodSelector(#selector(UIScreen.stubMainScreen), withSelector: #selector(UIScreen.mainScreen), forClass: UIScreen.self)
+    }
+
+    /// Setup the UIScreen for the given test
+    func setupScreen(isPortrait isPortrait: Bool, horizontalSizeClass: UIUserInterfaceSizeClass, verticalSizeClass: UIUserInterfaceSizeClass) {
+        StubScreen.horizontalSizeClass = horizontalSizeClass
+        StubScreen.verticalSizeClass = verticalSizeClass
+        setScreenPortrait(isPortrait)
+    }
+
+    func setScreenPortrait(isPortrait: Bool) {
+        StubScreen.bounds.size = isPortrait ? CGSize(width: 320, height: 480) : CGSize(width: 480, height: 320)
+    }
+
+}
+
+
+// Mark: - Screen
+
+extension UIScreen {
+    @objc class func stubMainScreen() -> UIScreen {
+        return StubScreen()
+    }
+}
+
+class StubScreen: UIScreen {
+    static var horizontalSizeClass: UIUserInterfaceSizeClass = .Unspecified
+    static var verticalSizeClass: UIUserInterfaceSizeClass = .Unspecified
+    static var bounds: CGRect = CGRect(x: 0, y: 0, width: 320, height: 480)
```

Comment:
this should have a reset method that gets called in `tearDown` so that some test runs don't potentially affect others.